wallet.md

```diff
@@ -0,0 +1,133 @@
+Wallet API
+==========
+
+BlockAdded Notifications
+------------------------
+```
+type NotifyBlockAddedRequestMessage struct {}
+
+type NotifyBlockAddedResponseMessage struct {
+	Error *RPCError
+}
+
+type BlockAddedNotificationMessage struct {
+	Block *RPCBlock
+}
+```
+ChainChanged Notifications
+--------------------------
+```
+type NotifyChainChangedRequestMessage struct {}
```

Comment:
Note: this will be useless without BlockAdded notifications, so don't remove them.